reflex/components/radix/primitives/accordion.py

```diff
@@ -193,6 +193,11 @@ class AccordionItem(AccordionComponent):
     # When true, prevents the user from interacting with the item.
     disabled: Var[bool]
 
+    # The header of the accordion item.
+    header: Optional[Union[Component, str, Var[Union[Component, str]]]] = None
+    # The content of the accordion item.
+    content: Optional[Union[Component, str, Var[Union[Component, str]]]] = None
```

Comment:
```suggestion
    # The header of the accordion item.
    header: Var[Union[Component, str]] = None
    # The content of the accordion item.
    content: Var[Union[Component, str]] = None
```

I think this is okay for these. If a `Var` is acceptable, then we can rely on the pyi generator to splat out the inner types